src/views/MondoMarket/index.jsx

```
@@ -0,0 +1,228 @@
+import {
+  getMondoByCategory,
+  getMondoMarketCategories,
+  getTopMondoMarket,
+} from "../../app/features/mondomarket";
+
+const MondoMarket = () => {
+  const [searchQuery, setSearchQuery] = useState("");
+
+  const [addModal, setAddModal] = useState(false);
+  const [mondoModal, setMondoModal] = useState(false);
+  const [reload, setReload] = useState(false);
```

ESLint

[error] 'reload' is assigned a value but never used.

(no-unused-vars)